jinja2/filters.py

```diff
@@ -967,6 +969,13 @@ def do_map(*args, **kwargs):
 
         Users on this page: {{ users|map(attribute='username')|join(', ') }}
 
+    If the list of objects does not contain the given attribute
+    you can specify a default value to be returned.
+
+    .. sourcecode:: jinja
+
+        Users on this page: {{ users|map(attribute='username', default='foobar')|join(', ') }}
```

Comment:
"Anonymous" might make more sense for this example.

Reply:
good idea, fixed in most recent commit.